src/VerifyCustomFactorController.js

```diff
@@ -108,6 +133,18 @@ export default FormController.extend({
     };
   },
 
+  postRender() {
+    if (this.settings.get('features.skipIdpFactorVerificationBtn')) {
+      if (this.options.appState.get('lastFailedChallengeFactorData')) {
```

Comment:
nit: I don't think this `if` is needed as is. In `formChildren` above, you add the spinner only if `skipIdpFactorVerificationBtn=true` and `!lastFailedChallengeFactorData`. So here in postRender we only need to hide the button bar and show the spinner of those 2 conditions are met, otherwise, there are no changes to what we currently have
```
if (this.settings.get('features.skipIdpFactorVerificationBtn') && !this.options.appState.get('lastFailedChallengeFactorData')) {
  this.$('.o-form-button-bar').hide();
  this.$('.okta-waiting-spinner').show();
}
```
right?